projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -405,23 +405,67 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
     }
 
     /**
-     * @hidden
+     * Emitted when the column is hidden or shown.
+     *
+     * ```html
+     * <igx-column (hiddenChange)="hiddenChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
      */
     @Output()
     public hiddenChange = new EventEmitter<boolean>();
 
-    /** @hidden */
+    /**
+     * Emitted when the column expanded or collapsed.
+     *
+     * ```html
+     * <igx-column (expandedChange)="expandedChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
+     */
     @Output()
     public expandedChange = new EventEmitter<boolean>();
 
-    /** @hidden */
+    /**
+     * Emitted when the column property `collapsible` changes.
+     *
+     * ```html
+     * <igx-column (collapsibleChange)="collapsibleChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
+     */
     @Output()
     public collapsibleChange = new EventEmitter<boolean>();
-    /** @hidden */
+
+    /**
+     * Emitted when the column property `visibleWhenCollapsed` changes.
+     *
+     * ```html
+     * <igx-column (visibleWhenCollapsedChange)="visibleWhenCollapsedChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
+     */
     @Output()
     public visibleWhenCollapsedChange = new EventEmitter<boolean>();
 
-    /** @hidden */
+    /**
+     * Emitted when the column changed.
+     *
+     * ```html
+     * <igx-column (columnChange)="columnChange($event)">
+     * </igx-column>
+     * ```
+     *
+     * @memberof IgxColumnComponent
+     */
     @Output()
     public columnChange = new EventEmitter<void>();
```

Comment:
Same here, can't even decide based on the description what this event is even supposed to be. Perhaps this is meant to be `@internal` as well instead?

Reply:
ok